### ISSUE TITLE
Child of the Plains + Talisman of the Sun

### ISSUE THREAD

**Issue by mellchia** [gryffon/ringteki#3197]
Glitch occurred when opponent's Child of the Plains (+ others) attacking a province. Reaction on the Child of the Plains was not triggered. My Talisman of the Sun Action moves the conflict to another province - province revealed. Child triggers Reaction, gives opponent first action...but that should have no effect. Opponent gets Action and then got another Action again. I never got my Action window.

TLDR: Moving conflict to a new province allowed Child of the Plains to react and give opponent two Actions in a row (both passing). Conflict ended after Opponent passed twice.

![ChildTalismanError](https://user-images.githubusercontent.com/50726846/57901183-d0d9e580-7818-11e9-8be4-4b38b5288f99.png)

**Comment by jamesoddy** [comment]
Child of the Plains should not be allowed to trigger his ability because as you said it should have no effect.  Ok, we will take a look - thanks for the report.

**Comment by jamesoddy** [comment]
Confirmed the problem with a test and the above PR has been created to fix the issue.  This issue should auto close once the PR is merged.

Thanks again for the report and particularly the screenshot of the chat - screenshots are especially useful!
